### PR TITLE
agoric-cli README clarification

### PR DESCRIPTION
**Pull request by michaelfig**
Doc fixes.

### CODE REVIEW

**Review by Chris-Hibbert — APPROVED** [review]
Looks good. Thanks, Michael!

File: packages/agoric-cli/README.md [hunk 1]
```diff
@@ -1,56 +1,24 @@
...
-
-If you have cloned and installed the Agoric SDK as described by its [README](/Agoric/agoric-sdk/#readme), you can use the Agoric CLI directly.
+If you want to modify the `template` directory used by Agoric CLI, you can run:
 
 ```sh
-alias agoric="/PATH/TO/agoric-sdk/packages/agoric-cli/bin/agoric --sdk"
+# Change to the template directory
+cd template
+# Install any needed links
+../bin/agoric --sdk install
```
Comment (katelynsills) [inline packages/agoric-cli/README.md]: What does this do? I get this error:

agoric: ERROR: [Error: EEXIST: file already exists, symlink '/Users/katelynsills/code/agoric-sdk/node_modules' -> 'node_modules'] {
  errno: -17,
  code: 'EEXIST',
  syscall: 'symlink',
  path: '/Users/katelynsills/code/agoric-sdk/node_modules',
  dest: 'node_modules'
}

**Review by katelynsills — APPROVED** [review]
Looks good to me! I had a question about the install step.

File: packages/agoric-cli/README.md (hunk 1 above)
Comment (michaelfig) [reply packages/agoric-cli/README.md]: Mea culpa!  You shouldn't need this step.  I'll remove it.

**Pull request merged**